tests/eosio_spec_test_generator/altered-wasms/memory_size/notes.md

```diff
@@ -0,0 +1,7 @@
+## Changes
```

Comment:
Why did we make these changes?

Reply:
When we build the unit test, the `eosio::check` messages are added to the end of the WASM as data segments. So we need at least one page of memory or we get an error for the data segment being out of memory.